AutoGVP/filter_vcf.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+vcf_file=$1
+exp_args=("$@")
+
+## get vcf filename
+vcf_filtered_file=$1."filtered.vcf"
```

Comment:
```suggestion
vcf_filtered_file=${vcf_file%.vcf*}."filtered.vcf"
```

This will remove the `.vcf` or `.vcf.gz` suffix before adding the new file suffix